resources/keb/templates/rbac.yaml

```diff
@@ -7,24 +7,21 @@ metadata:
     app: {{ include "kyma-env-broker.name" . }}
     release: {{ .Values.namePrefix }}
 rules:
-  - apiGroups: ["*"]
+  - apiGroups: [""]
     resources: ["secrets"]
-    verbs: ["*"]
-  - apiGroups: ["*"]
+    verbs: ["get", "list"]
+  - apiGroups: [""]
     resources: ["configmaps"]
     verbs: ["list", "get"]
-  - apiGroups: ["core.gardener.cloud"]
```

Comment:
this service account and role is for KCP, shoots and secret bindings are defined in the gardener (different K8S cluster)